### PR TITLE
Updates `BTrDB.info()` to return a dict

### PR DESCRIPTION
**Pull request by looselycoupled**
This PR changes `BTrDB.info()` to return a dict rather than the original `btrdb_pb2.InfoResponse`.

Also included:
- related tests
- updates to docs (including unrelated changes)
- adds missing docs file for API section

### CODE REVIEW

File: btrdb/conn.py [hunk 1]
```diff
@@ -159,7 +159,7 @@ def stream_from_uuid(self, uuid):
         Returns
         -------
         Stream
```
Comment (bbengfort) [inline btrdb/conn.py]: In the doc style we use, I generally shorten this to 

```
Returns
--------
Stream : instance of Stream class or None if not exists
```

Does alabaster (that's what you're using right?) not support this format?

**Comment by looselycoupled** [comment]
[ch69]

File: tests/btrdb/test_conn.py [hunk 2]
```diff
@@ -88,3 +89,22 @@ def test_streams_stores_versions(self):
 
         streams = db.streams(uuid1, uuid2, versions=versions)
         assert streams._pinned_versions == expected
+
+    def test_info(self):
```
Comment (bbengfort) [inline tests/btrdb/test_conn.py]: nice

File: btrdb/conn.py [hunk 3]
```diff
@@ -200,11 +200,16 @@ def info(self):
...
         Returns
         -------
-        InfoResponse
-            An object containing server connection and status information
+        dict
+            server connection and status information
 
         """
-        return self.ep.info()
+        info = self.ep.info()
```
Comment (bbengfort) [inline btrdb/conn.py]: was it returning a string before? guess there was no way of json.loads it?

Comment (looselycoupled) [reply btrdb/conn.py]: it used to return `btrdb_pb2.InfoResponse` which wasnt as helpful as a basic `dict`

File: btrdb/conn.py (hunk 1 above)
Comment (looselycoupled) [reply btrdb/conn.py]: I just tested and  the one liner you refer to also works in Alabaster.  Will update them as I resolve issues.

File: btrdb/conn.py (hunk 3 above)
Comment (bbengfort) [reply btrdb/conn.py]: ok, well if this changes in the future; let's try to remind ourselves to look into the proto code - in go there is a way to convert the packed data into JSON; hopefully there is something similar in Python?
